dev/TeachingTip/TeachingTip.xaml

```diff
@@ -561,11 +561,19 @@
 
                                                     <Border x:Name="IconBorder" Grid.Column="0" Child="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.IconElement}"/>
                                                     <StackPanel x:Name="TitlesStackPanel" Grid.Column="1">
-                                                        <TextBlock x:Name="TitleTextBlock" Grid.Column="0" Text="{TemplateBinding Title}" TextWrapping="WrapWholeWords" FontWeight="SemiBold"/>
-                                                        <TextBlock x:Name="SubtitleTextBlock" Grid.Row="1" Text="{TemplateBinding Subtitle}" TextWrapping="WrapWholeWords"/>
+                                                        <TextBlock x:Name="TitleTextBlock" Grid.Column="0" Text="{TemplateBinding Title}" TextWrapping="WrapWholeWords"
+                                                            Foreground="{TemplateBinding Foreground}" FontFamily="{TemplateBinding FontFamily}"
+                                                            FontWeight="SemiBold"/>
+                                                        <TextBlock x:Name="SubtitleTextBlock" Grid.Row="1" Text="{TemplateBinding Subtitle}"
+                                                            Foreground="{TemplateBinding Foreground}" FontFamily="{TemplateBinding FontFamily}"
```

Comment:
>Foreground [](start = 89, length = 10)

This means there is no way to have the title and the subtitle be different colors without retemplating.. :/

Reply:
Yup, Should we create lightweight styling resources for that or even create new properties on the TeachingTip?

Reply:
Lightweight styling sounds reasonable to me.

Reply:
@StephenLPeters Are you also in favor of lightweight styling resources for that?

Reply:
I think so, its the pattern we have.. @ranjeshj and @MikeHillberg

Reply:
Added them now.